vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1431,84 +1400,71 @@ public void fetchCurrentUser(@NotNull ModelCallback<User> callback, @Nullable St
      * @param query         Query string for hitting the search endpoint
      * @param refinementMap Used to refine lists (generally for search) with sorts and filters
      * @param fieldFilter   The string of fields to include in the response (highly recommended!)
-     *                      {@link SearchRefinementBuilder}
+     *                      {@link RequestRefinementBuilder}
+     * @param caller        The {@link Caller} for the expected response type
      * @see <a href="https://developer.vimeo.com/api/spec#common-parameters">Vimeo API Field Filter Docs</a>
      */
     @Nullable
-    public Call<Object> fetchContent(@NotNull String uri, CacheControl cacheControl,
-                                     @NotNull ModelCallback callback, @Nullable String query,
-                                     @Nullable Map<String, String> refinementMap,
-                                     @Nullable String fieldFilter) {
+    public <DataType_T> Call<DataType_T> getContent(@NotNull String uri,
+                                                              @NotNull CacheControl cacheControl,
+                                                              @NotNull VimeoCallback<DataType_T> callback,
+                                                              @Nullable String query,
+                                                              @Nullable Map<String, String> refinementMap,
+                                                              @Nullable String fieldFilter,
+                                                              @NotNull Caller<DataType_T> caller) {
```

Comment:
It appears as though the formatting is a little wonky on this file (after pulling it down).